parsers/lib/config.py

```diff
@@ -19,19 +23,19 @@ def __str__(self) -> str:
         return self.value
 
     @classmethod
-    def values(cls) -> list[str]:
-        """Return a list of all enum string values."""
-        return [mode.value for mode in cls]
+    def values(cls: type[ModeEnumType]) -> ValuesView[ModeEnumType]:
```

Comment:
Should this be renamed to members?

Reply:
Tough call, but I think values is better, people are more used to dict and it's methods, than enum functionality. So hiding this and keeping a more dict-like interface is nice